devsite/devsite/urls.py

```diff
@@ -2,16 +2,17 @@
 URL patterns includes for Figures devsite
 '''
 
+from __future__ import absolute_import
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.conf import settings
 from django.views.generic import TemplateView
 
 urlpatterns = [
     url(r'^$', TemplateView.as_view(template_name='homepage.html'), name='homepage'),
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', admin.site.urls),
     url(r'^accounts/', include('django.contrib.auth.urls')),
-    url(r'^figures/', include('figures.urls', namespace='figures')),
+    url(r'^figures/', include(('figures.urls', 'figures'), namespace='figures')),
```

Comment:
Learning: What's the first argument (the tuple) means? in `include(('figures.urls', 'figures')`